src/main/java/SYSC4806/HomeController.java

```diff
@@ -0,0 +1,11 @@
+package SYSC4806;
```

Comment:
I think we should also have some controllers package. Might as well add it on this feature since it's using the first controller -shrug-